app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -480,7 +480,7 @@ public boolean onLongClick(View v) {
                             checked = i;
                         }
                         choices[i] = String.valueOf(values[i]) + " "
-                                + getString(R.string.time_unit_seconds);
+                                + getString(R.string.time_seconds);
```

Comment:
Might be able to work around that by just removing the number from the
string. (though perhaps that would cause it to be wrong in some languages)

On Sun, Aug 2, 2015, 6:48 PM Tom Hennen <tom.hennen@gmail.com> wrote:

> When it's used for the sleep timer having the integer (1 or d) doesn't
> make sense with the way it's displayed to the user.
>
> On Sun, Aug 2, 2015, 6:38 PM Martin Fietz <notifications@github.com>
> wrote:
>
>> In
>> app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java
>> <https://github.com/AntennaPod/AntennaPod/pull/1055#discussion_r36049097>
>> :
>>
>> > @@ -480,7 +480,7 @@ public boolean onLongClick(View v) {
>> >                              checked = i;
>> >                          }
>> >                          choices[i] = String.valueOf(values[i]) + " "
>> > -                                + getString(R.string.time_unit_seconds);
>> > +                                + getString(R.string.time_seconds);
>>
>> e.g. choices[i] =
>> context.getResources().getQuantityString(R.time_unit_seconds, values[i],
>> values[i]);
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/AntennaPod/AntennaPod/pull/1055/files#r36049097>.
>>
>